src/BloomExe/ImageProcessing/ImageServer.cs

```diff
@@ -104,5 +105,14 @@ protected override bool ProcessRequest(IRequestInfo info)
 			}
 			return false;
 		}
+
+		protected static bool IsImageTypeThatCanBeDegraded(string path)
```

Comment:
The test here seems to me to have nothing to do with whether the image can be degraded. It has to do with whether it can be found at all. The image server doesn't even try to find anything that isn't one of these file types. (The EnhancedImageServer subclass we actually use may, of course.)